sdl_viewer/src/main.rs

```diff
@@ -421,6 +439,9 @@ fn main() {
                 camera.height,
                 octree::UseLod::Yes,
             );
+            if visible_nodes.len() > cache_size {
+                visible_nodes.truncate(cache_size);
```

Comment:
only truncate, no need for the if-statement. See docs https://doc.rust-lang.org/std/vec/struct.Vec.html#method.truncate